package.json

```diff
@@ -43,40 +43,41 @@
         "@nrwl/workspace": "^12.10.0",
         "@types/express": "^4.17.13",
         "@types/jest": "^27.0.2",
+        "@types/node": "*",
+        "@types/node-fetch": "^2.5.12",
+        "@types/node-jose": "^1.1.10",
         "@types/oidc-provider": "^7.6.1",
         "@types/pino": "^6.3.11",
         "@types/supertest": "^2.0.11",
-        "@types/node-fetch": "^2.5.12",
         "@typescript-eslint/eslint-plugin": "4.22.0",
-        "@typescript-eslint/parser": "4.22.0",
+        "@typescript-eslint/parser": "~4.31.1",
         "@wanews/nx-typescript-project-references": "^0.14.0",
+        "@types/aws-lambda": "^8.10.83",
+        "@types/cookie": "^0.4.1",
+        "@types/jsonwebtoken": "^8.5.5",
         "babel-jest": "^27.2.1",
         "cuid": "^2.1.8",
         "esbuild": "^0.11.12",
         "eslint": "7.24.0",
         "eslint-config-prettier": "8.2.0",
         "express": "^4.17.1",
         "jest": "^27.2.1",
+        "node-jose": "^2.1.0",
```

Comment:
I think this might be needed to be added to the `lambda-edge-openid-auth` lib `package.json` as a dependency. I had to add in `jwk-to-pem` in there because it threw out an error when trying to use the package.